lib/di.dart

```diff
@@ -768,6 +828,16 @@ Future<void> setup({
   getIt.registerFactoryParam<WalletRestorePage, WalletType, void>(
       (type, _) => WalletRestorePage(getIt.get<WalletRestoreViewModel>(param1: type)));
 
+  getIt.registerFactoryParam<WalletRestoreChooseDerivationViewModel, List<DerivationInfo>, void>(
+      (derivations, _) =>
+          WalletRestoreChooseDerivationViewModel(derivationInfos: derivations));
+
+  getIt.registerFactoryParam<WalletRestoreChooseDerivationPage, dynamic, void>(
```

Comment:
that's why I don't like dynamic.
so you are passing a `dynamic` parameter called `credentials` and is expecting a `List<DerivationInfo>` does that make sense?